news/2018-04-08-project-loved-week-of-april-8th.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: post
+title: "Project Loved: Week of April 8th"
+date: 2018-04-08 18:00:00 +0000
+---
+
+Another Sunday marks another [Project Loved](https://osu.ppy.sh/community/forums/120) voting on the horizon! Come and vote for your favourite maps nominated from all four game modes!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Hello and welcome to the first [Project Loved](https://osu.ppy.sh/community/forums/120) showcase of April 2018! This week contains several exciting picks for every game mode, and has something for both new and old players!
+
+For those of you who are unfamiliar with Project Loved or how the new system works, each mode has "captains" who pick a selection of your submissions to be put up for vote every two weeks. These chosen selections are each given their own community poll, and any map with an approval rating above 75% after a week's time gets moved into the Loved category. Maps that are Loved have leaderboards, but don't award performance points (much like Qualified maps).
```

Comment:
gets -> is
Maps -> Beatmaps
don't -> do not
much like -> very much like

Reply:
why add "very"?